addons/html_editor/static/tests/power_buttons.test.js

```diff
@@ -20,6 +21,26 @@ describe("visibility", () => {
         expect(".o_we_power_buttons").not.toBeVisible();
     });
 
+    test("should show power buttons on block p tag contains strong (bold)", async () => {
```

Comment:
```suggestion
    test("should show power buttons on P tag containing strong (bold)", async () => {
```
same for all below test cases.